packages/application-components/src/experimental/components/product-picker-input/product-picker-input.spec.tsx

```diff
@@ -0,0 +1,216 @@
+import type { TRenderAppOptions } from '@commercetools-frontend/application-shell/test-utils';
+
+// import { graphql } from 'msw';
+// import { setupServer } from 'msw/node';
+import React from 'react';
+import {
+  renderApp,
+  fireEvent,
+  screen,
+} from '@commercetools-frontend/application-shell/test-utils';
+import { GRAPHQL_TARGETS } from '@commercetools-frontend/constants';
+import ProductPickerInput from './product-picker-input';
+import FetchProductByPrefix from './fetch-product-by-prefix.ctp.graphql';
+import PrefetchProduct from './prefetch-selected-product.ctp.graphql';
+
+const productId = '6f69c022-b894-4deb-b19f-6335547cb5d7';
+const inputName = 'product-picker-input';
+
+const createTestProps = (custom = {}) => ({
+  name: inputName,
+  value: productId,
```

Comment:
Better maybe to consider the default props without a `value`. Then when you want to test with an existing value, you pass it explicitly.